content/slice/language-guide/result-types.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Result types
+description: Learn how to define and use results in Slice.
+---
+
+{% slice1 %}
+The built-in generic type Result<Success, Failure> is not available with Slice1.
```

Comment:
Again, I think it's good to wrap this in quotes, since it isn't a single word like 'sequence' or 'dictionary'.
It's a programmatic expression, not a grammatical expression.
```suggestion
The built-in generic type 'Result<Success, Failure>' is not available with Slice1.
```

Reply:
Fixed as suggested